docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-edge-syntax.md

```diff
@@ -1,6 +1,6 @@
 # Delete Edge Syntax
 
-Given an edge type, the source vertex and the dest vertex, Nebula supports `DELETE` the edge, its associated properties, the edge ranking, and edge with a certain rank, the syntax as the follows:
+The `DELETE EDGE` statement is used to delete edges. Given an edge type, the source vertex and the dest vertex, Nebula supports `DELETE` the edge, its associated properties and the edge ranking. You can also delete an edge with a certain rank. The syntax as the follows:
```

Comment:
Nebula -> Nebula Graph

Reply:
The syntax as the follows -> The syntax is as follows:

Reply:
> Nebula -> Nebula Graph
Will open a new pr to unify all the name to `Nebula Graph`.